dataset_api/search.py

```diff
@@ -222,6 +222,8 @@ def delete_data(id):
 
 
 def facets(request):
+    size = request.GET["size"]
+    frm = request.GET["from"]
```

Comment:
rename variable to from

Reply:
from is pythonic keyword, so it won't work as a variable.

Reply:
Right. else use another readable name like results_from or any other which makes sense in context.